html5/js/VideoDecoder.js

```diff
@@ -99,13 +100,14 @@ class XpraVideoDecoder {
 
     // Latest possible check for draining
     if (this.draining) {
+
```

Comment:
left over CR?

Reply:
Yes...fixed in #200 .
Is backporting to this PR needed?

Reply:
No, let's move forward only.